### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by LethaboMot**
in the project I added:

Added CSS
Added javascript
Added images

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -15,6 +29,7 @@ const projects = [{
   description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since th',
   image: 'img/Nature',
   technologies: ['html', 'JavaScript', 'css'],
+  source: 'https://github.com/LethaboMot',
```
Comment (tufoinnkuo10) [inline popup.js:32]: - [ ] nice work. please kindly ensure that from mobile screen size of `375` the `seesource` and `seelive `of the mobile version should be well aligned like that of figma version.
yours
Screenshot (698)_LI

File: popup.js [hunk 2]
```diff
@@ -15,6 +29,7 @@ const projects = [{
...
   link: 'https://lethabomot.github.io/Port-folio/',
 },
 {
```
Comment (tufoinnkuo10) [inline popup.js:35]: - [ ] please kindly align the link buttons, so it matches the original design for the desktop version
- [ ] kindly put the technologies above the links and align to the right as well. nice work
yours
Screenshot (699)_LI
figma 

Screenshot (691)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @LethaboMot 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile popup works as expected


Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by mikethreels — APPROVED** [review]
Hi Lethabo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**